src/components/Navbar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'BOOKS',
+    },
+    {
+      id: 2,
+      path: '/categories',
+      text: 'CATEGORIES',
```

Comment:
- [ ] When I click the categories link, this should show in the application the text `Under construction` text only, but I'm afraid this doesn't appear, please, work on that. 🐱 
![image](https://user-images.githubusercontent.com/20563252/130534144-21cfa05d-fba0-4de7-92d1-943f88907b88.png)